### ISSUE TITLE
Releases and development branch

### ISSUE THREAD

**Issue by MichaIng** [HTPC-Manager/HTPC-Manager#40]
Just an idea to create a `dev` branch to accumulate some commits to not cause an update notification and trigger for possible rebuilds on every little PR/commit merged.

We could combine this with release tags, but that's more optional as currently the code only cares about commits, not about version strings or similar at all.

**Comment by MichaIng** [comment]
Development branch created from master2: https://github.com/HTPC-Manager/HTPC-Manager/tree/dev
I'll open/rebase all PRs onto it. This way we have a chance to merge and test multiple PRs and then review and release everything combined.

**Comment by gmiranda** [comment]
:+1: 

Perhaps we can consider tagging a release when support for a new OS/distro/language version is merged? Besides that, you're idea sounds practical to me, I'm buying.

**Comment by MichaIng** [comment]
Good idea, or when dropping support for old OS/distro/language version or other kind of breaking change.
